CLAUDE.md

```
@@ -0,0 +1 @@
+Before working on anything in this repository, read and follow the @AGENTS.md file.
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)